python/uv/__main__.py

```diff
@@ -31,6 +31,9 @@ def _run() -> None:
     if venv:
         env.setdefault("VIRTUAL_ENV", venv)
 
+    # Let `uv` know that it was spawned by this Python interpreter
+    env["UV__PARENT_INTERPRETER"] = sys.executable
```

Comment:
Can we use something other than the double underscore, like a name that is so long that no one likes to use it outside that script?

Reply:
I feel like the double underscore is sufficient for saying it's not user facing and we're not going to document or support it elsewhere. Idk about making the name arbitrarily longer.... I guess I could do `UV_PRIVATE__...`?

Reply:
Added more suffix